File: PIM_Tool_ELCA/client/src/app/project-list/project-table/model/project-list-response-model.ts
export class ProjecListResponseModel {
  public ProjectList: any[];
  public ResultCount: number;
  public PageIndex: number;
  public PageSize: number;
  constructor(ProjectList: any[],
    ResultCount: number,
    PageSize: number,
    PageIndex: number,
  ) {
    this.ProjectList = ProjectList;
    this.ResultCount = ResultCount;
    this.PageSize = PageSize;
    this.PageIndex = PageIndex;
  }
}

File: ContractLayer/AngularResponseModel/ProjectListResponseModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ContractLayer.AngularResponseModel
{
    public class ProjectListResponseModel

    {
        public List<ProjectListModel> ProjectList { get; set; }
        public int ResultCount { get; set; }
        public int PageIndex { get; set; }
        public int PageSize { get; set; }
    }
}
